### PR TITLE
Task1 added guestlist class

### PR DESCRIPTION
**Pull request by cvintilianrosca**

### CODE REVIEW

File: play-with-git/src/Lalal.java [hunk 1]
```diff
@@ -1,5 +1,13 @@
+import java.util.ArrayList;
+
 public class Lalal {
 
+    public static void main(String[] args) {
+        ArrayList<String> s = new ArrayList<>(10);
+        System.out.println(s.size());
+        s.add("erfwken");
+        System.out.println(s.size());
```
Comment (cvintilianrosca) [inline play-with-git/src/Lalal.java:9]: I think this changes are not related to the task. Maybe to revert them?

File: play-with-git/src/w2/sistem_inscrieri/Guest.java [hunk 2]
```diff
@@ -37,4 +40,17 @@ public String getPhoneNumber() {
...
         this.phoneNumber = phoneNumber;
     }
+
+    @Override
+    public boolean equals(Object object) {
+        if (this == object) return true;
+        if (object == null || getClass() != object.getClass()) return false;
+        Guest guest = (Guest) object;
+        return Objects.equals(lastName, guest.lastName) && Objects.equals(firstName, guest.firstName) || Objects.equals(email, guest.email) || Objects.equals(phoneNumber, guest.phoneNumber);
+    }
```
Comment (cvintilianrosca) [inline play-with-git/src/w2/sistem_inscrieri/Guest.java:50]: OK...

File: play-with-git/src/w2/sistem_inscrieri/GuestList.java [hunk 3]
```diff
@@ -0,0 +1,103 @@
+package w2.sistem_inscrieri;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+
+public class GuestList implements GuestListActions {
+    private final int locuriDisponibile;
+    private ArrayList<Guest> participanti;
+    private LinkedList<Guest> inAsteptare;
```
Comment (cvintilianrosca) [inline play-with-git/src/w2/sistem_inscrieri/GuestList.java:10]: Maybe to use a combination of a hashMap and a List to improve the get performance or something similar? 
What do you think @alexandruzarnoianu  about it?

File: play-with-git/src/w2/sistem_inscrieri/GuestList.java [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+
+    public List<Guest> getParticipanti() {
+        return participanti;
+    }
+
+//    public void setParticipanti(List<Guest> participanti) {
+//        this.participanti = participanti;
+//    }
+
```
Comment (cvintilianrosca) [inline play-with-git/src/w2/sistem_inscrieri/GuestList.java:30]: Please remove the commented code

File: play-with-git/src/w2/sistem_inscrieri/GuestList.java [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+        guestToUpdate.setPhoneNumber(guest.getPhoneNumber());
+    }
+
+    private void printConfirmParticipant(Guest guest) {
+        System.out.format("%s %s Felicitari! Locul tau la eveniment este confirmat. Te asteptam!.\n", guest.getLastName(), guest.getFirstName());
+    }
+
+    private void printConfirmInAsteptare(Guest guest) {
+        System.out.format("%s %s Te-ai inscris cu succes in lista de asteptare si ai primit numarul de ordine <%d>. Te vom notifica daca un loc devine disponibil.\n", guest.getLastName(), guest.getFirstName(), inAsteptare.indexOf(guest));
+    }
```
Comment (cvintilianrosca) [inline play-with-git/src/w2/sistem_inscrieri/GuestList.java:102]: This code does not look formatted

**Review by alexandruzarnoianu — APPROVED** [review]
